plugins/laravelforge/forge.go

```diff
@@ -0,0 +1,28 @@
+package laravelforge
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func LaravelForgeCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Laravel Forge CLI",
+		Runs:    []string{"forge"},
```

Comment:
Similar to https://github.com/1Password/shell-plugins/pull/245/files#r1166566495

Let me know what you think about using `php vendor/bin/forge` instead, here.

Reply:
The Forge CLI is only meant to be installed **globally** via [Composer](https://getcomposer.org)(the de-facto PHP package manager) as stated in their docs.

As far as I'm aware there are no plans to make it installable on a per project bases (correct me if I'm wrong @driesvints, @nunomaduro)

After globally installing the Forge CLI via [Composer](https://getcomposer.org) the `forge` command is available as `forge` in your command line assuming a proper [Composer](https://getcomposer.org) installation and a configured $PATH on Unix systems.

Relevant links on how this works:
https://getcomposer.org/doc/00-intro.md#globally
https://akrabat.com/global-installation-of-php-tools-with-composer/

Btw this flow also works for [Composer installed via Homebrew(Mac)](https://formulae.brew.sh/formula/composer) because most likely the Homebrew installation is added to the $PATH which makes Homebrew-installed programs available in the shell. This is already done on macOS 10.14 Mojave and newer by default. I would even argue that this setup is the most commonly used at least on macOS.

Hopefully, this clarifies why I've chosen `forge` instead of `php vendor/bin/forge`